api/Dockerfile

```diff
@@ -17,45 +17,34 @@
 
 FROM golang:1.13.8 AS build-env
 
-WORKDIR /go/src/github.com/apisix/manager-api
+RUN mkdir -p /usr/local/apisix-dashboard/api
```

Comment:
one PR for one thing, I think we can fix this in new PR

Reply:
ok,let's revert this file